### PR TITLE
P4 885 part 3 cleanup

### PR DESCRIPTION
**Pull request by starswan**

### CODE REVIEW

**Comment by martyn-w** [comment]
@starswan please pull in the latest master branch

**Comment by martyn-w** [comment]
@starswan can you pull in the latest master branch?

File: db/migrate/20200211105449_remove_unused_person_id_from_move.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class RemoveUnusedPersonIdFromMove < ActiveRecord::Migration[5.2]
+  def up
+    remove_column :moves, :person_id
+  end
+  def down
+    change_table :moves do |t|
+      t.uuid :person_id
+      t.index ["from_location_id", "to_location_id", "person_id", "date"], name: "index_on_move_uniqueness", unique: true
```
Comment (alexdesi) [inline db/migrate/20200211105449_remove_unused_person_id_from_move.rb]: Why do you add index to 
from_location_id, to_location_id and date
hen you rollback the migration?

**Comment by martyn-w** [comment]
@starswan is this PR still required? if not, can we close it?

**Comment by martyn-w** [comment]
Blocked until pre-prod and production are synchronised

**Pull request closed without merging**